website/docs/components/time/partials/code/how-to-use.md

```diff
@@ -0,0 +1,43 @@
+!!! Info
+
+The Time component exists only in code. The Figma equivalent would be a manually input text object with a formatted string.
+
+!!!
+
+## How to use this component
+
+The `Time` component is used to display dates and times in a consistent format.
```

Comment:
This should probably have the backticks removed to match the other mentions of the component.

```suggestion
The Time component is used to display dates and times in a consistent format.
```